website/docs/r/windows_virtual_machine.html.markdown

```diff
@@ -141,6 +141,8 @@ The following arguments are supported:
 
 -> **NOTE:** This can only be configured when `priority` is set to `Spot`.
 
+* `patch_mode` - (Optional) Specifies the mode of in-guest patching to this Windows Virtual Machine. Possible values are `Manual`, `AutomaticByOS` and `AutomaticByPlatform`. Defaults to `AutomaticByOS`.
```

Comment:
Could we include a note that this is a preview feature and the command to enable it?
```
 TestAccWindowsVirtualMachine_otherPatchModeUpdate: testing.go:684: Step 2 error: errors during apply:
        
        Error: updating Windows Virtual Machine "acctestvmlv1eo" (Resource Group "acctestRG-201114233915712563"): compute.VirtualMachinesClient#Update: Failure sending request: StatusCode=400 -- Original Error: Code="InvalidParameter" Message="The property 'windowsConfiguration.patchSettings.patchMode' is not valid because the 'Microsoft.Compute/InGuestAutoPatchVMPreview' feature is not enabled for this subscription." Target="windowsConfiguration.patchSettings.patchMode"
```

Reply:
Oh...yeah I will update the doc to include the enabling process for this. Thanks